tinygrad/helpers.py

```diff
@@ -1,5 +1,5 @@
 from __future__ import annotations
-import os, functools, platform, time, re, contextlib, operator, hashlib, pickle, sqlite3, tempfile, pathlib, string, ctypes, sys
+import contextlib, ctypes, functools, gzip, hashlib, operator, os, pathlib, pickle, platform, re, sqlite3, string, sys, tempfile, time
```

Comment:
Why did this change?

Reply:
I sorted the imports alphabetically and added `gzip`.

Reply:
but why?

Reply:
I usually sort my imports.

LMK if you want me to do undo it.